execute/plugin_test.go

```diff
@@ -166,3 +155,496 @@ func Test_getPendingExecutedReports(t *testing.T) {
 		})
 	}
 }
+
+// TODO: better than this
+type tdr struct{}
+
+func (t tdr) ReadTokenData(
+	ctx context.Context, srcChain cciptypes.ChainSelector, num cciptypes.SeqNum) ([][]byte, error,
+) {
+	return nil, nil
+}
+
+// mustRandID generates a random hex ID value.
+func mustRandID() string {
+	bytes := make([]byte, 32)
+	if _, err := rand.Read(bytes); err != nil {
+		panic(err)
+	}
+	return hex.EncodeToString(bytes)[:32]
+}
+
+func breakCommitReport(
+	commitReport cciptypes.ExecutePluginCommitDataWithMessages,
+) cciptypes.ExecutePluginCommitDataWithMessages {
+	commitReport.Messages = append(commitReport.Messages, cciptypes.CCIPMsg{
+		CCIPMsgBaseDetails: cciptypes.CCIPMsgBaseDetails{
+			ID:          mustRandID(),
+			SourceChain: cciptypes.ChainSelector(1),
+			SeqNum:      cciptypes.SeqNum(999),
```

Comment:
nit: Would it make more sense if this isn't hardcoded and is passed in instead?

Reply:
I'll add a comment. It could probably just be an empty object. In this case it's breaking the commit report by adding an extra message to the report, the len(messages) != len(seqNumRange) check fails.